src/bit-systems/media-loading.ts

```diff
@@ -176,13 +180,16 @@ type MediaInfo = {
   thumbnail: string;
 };
 
+const loadingObjs = new Map<EntityID, EntityID>();
+
 function* loadMedia(world: HubsWorld, eid: EntityID) {
-  let loadingObjEid = 0;
   const addLoadingObjectTimeout = crTimeout(() => {
-    loadingObjEid = renderAsEntity(world, LoadingObject());
-    add(world, loadingObjEid, eid);
+    if (MediaLoader.flags[eid] & MEDIA_LOADER_FLAGS.ANIMATE_LOAD) {
+      const loadingObjEid = renderAsEntity(world, LoadingObject());
+      loadingObjs.set(eid, loadingObjEid);
+      add(world, loadingObjEid, eid);
+    }
```

Comment:
I thought `MEDIA_LOADER_FLAGS.ANIMATE_LOAD` is enable/disable `animateScale()` and is unrelated t`MEDIA_LOADER_FLAGS.ANIMATE_LOAD`o loading objects but with this change loading objects won't be displayed for non `MEDIA_LOADER_FLAGS.ANIMATE_LOAD`. But no confident about my assumption.

@johnshaughnessy Which is the expected behavior?

Reply:
Oh yeah, that's am error. Removing that check.